scripts/e2e-dryrun-creds.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/env python3
+import subprocess
+
+cmd = subprocess.run(['aws', 'configure', 'list-profiles'], stdout=subprocess.PIPE)
+profiles = cmd.stdout.decode('utf-8').split('\n')
+
+for e2eprofile in ['e2etestenv', 'e2eprodenv']:
+  if e2eprofile in profiles:
+    continue
+  print(f'Profile [{e2eprofile}] is required but does not exist. Copying your [default] access key id and secret key...')
```

Comment:
I wonder if we should just error out and prompt them to set up any missing profile and use different accounts/regions as possible for `e2etestenv` and `e2eprodenv`. Otherwise, even if they run the multiple env suite successfully within the same account/region, it doesn't mean multi-env scenario is ok 🤔

Reply:
I kept debating about this too! 

I'm not super sure because I want the dryrun to be easy to run and cover the majority of the test. Having to create 3 AWS accounts to do `make e2e-dryrun` seems cumbersome.  
If the engineer already has the `e2etestenv` and `e2eprodenv` profiles in separate accounts then we won't override those creds and the envs will be created in separate accounts 💭 only the e2e test won't check if they in separate accounts.

I'm down to change it though if folks prefer the prompt route